src/ansys/fluent/core/systemcoupling.py

```diff
@@ -44,7 +44,7 @@ class SystemCoupling:
     def __init__(self, solver):
         self._solver = solver
         # version check - this requires Fluent 2024 R1 or newer.
-        current_fluent_version = FluentVersion(self._solver.get_fluent_version())
+        current_fluent_version = FluentVersion(self._solver.get_fluent_version().value)
```

Comment:
`get_fluent_version()` is already returning a `FluentVersion`.

```suggestion
        current_fluent_version = self._solver.get_fluent_version()
```